vernac/metasyntax.ml

```diff
@@ -1006,7 +1006,15 @@ let subentry_of_constr_prod_entry = function
   | ETConstr (InConstrEntry,_,_) -> InConstrEntrySomeLevel
   | _ -> InConstrEntrySomeLevel
 
-let make_interpretation_vars recvars allvars typs =
+let warn_not_interpreting_binding =
+  CWarnings.create ~name:"not-interpreting-binding-variable" ~category:"parsing"
+         (fun id ->
+          strbrk "Bound variable " ++ Id.print id ++
+          str " not bound by the notation variable.")
+
```

Comment:
This new warning should be documented.

Reply:
Then I need help from @zimmi48 - I don't know where this documentation should go.

Reply:
It looks like this should be documented in the notation chapter: https://coq.inria.fr/refman/user-extensions/syntax-extensions.html#abbreviations
You should use the `.. warn::` directive to give the name of the warning.